devices/tuya.js

```diff
@@ -1696,6 +1708,24 @@ module.exports = [
         toZigbee: [],
         exposes: [e.battery(), e.temperature(), e.humidity(), e.battery_voltage()],
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE200_yjjdcqsq']),
+        model: 'ZTH01',
+        vendor: 'TuYa',
+        description: 'Temperature and humidity sensor',
+        fromZigbee: [tuya.fz.datapoints, fzLocal.gateway_connection_status],
+        toZigbee: [tuya.tz.datapoints],
+        configure: tuya.configureMagicPacket,
+        exposes: [e.temperature(), e.humidity(), tuya.exposes.batteryState(), e.battery_low()],
+        meta: {
+            tuyaDatapoints: [
+                [1, 'temperature', tuya.valueConverter.divideBy10],
+                [2, 'humidity', tuya.valueConverter.raw],
+                [3, 'battery_state', tuya.valueConverter.batteryState],
+                // [9, 'temperature_unit', tuya.valueConverter.raw], Does not have any effect on the device
```

Comment:
@Burki24 This line is  commented out, but the dp value 9 is added to lib/tuya.js
Why is that?

You write here in the comment "Does not have any effect on the device" -> What do you want to say with that?

Reply:
DP 9 stands for switching between °C and °F. But it's not really supported by this device. So also in the original APP it isn't usable. For that, we don`t need this DP inside the converter or as Expose.

Reply:
Ok, understood!
I'll suggest a change of the comment in the code, just in case someone else stumbles across that line in the future and asks the same question as I did.

One more question: Will the zigbee2mqtt log not show a "No converter available [...]" message for the DP 9, if we comment it out completely?
If so, maybe we should leave it in there and handle the useless data transmitted by it later, in order to avoid that log message

Reply:
No, I get no warnings or something else from the outcommented DP 9. So please, let it stay as it is.